scripts/mixins/families/aern.lua

```diff
@@ -11,6 +11,75 @@ g_mixins = g_mixins or {}
 g_mixins.families = g_mixins.families or {}
 
 g_mixins.families.aern = function(aernMob)
+    -- Aerns only spawn on engage, we need to manually control the spawn behavior with a summoning animation
+    local petSpawn = function(mob)
+        local mainJob = mob:getMainJob()
+
+        if mainJob == xi.job.SMN then
+            -- SMN Aerns only spawn 5 elementals
+            local eleCount = mob:getLocalVar('aernElementalCount')
+            if eleCount < 6 then
+                local elemental = math.random(xi.spell.FIRE_SPIRIT, xi.spell.DARK_SPIRIT)
+                mob:castSpell(elemental)
+                mob:setLocalVar('aernElementalCount', eleCount + 1)
+            end
+        elseif mainJob == xi.job.BST or mainJob == xi.job.DRG then
+            mob:entityAnimationPacket(xi.animationString.CAST_SUMMONER_START)
+            mob:setAutoAttackEnabled(false)
+            mob:setMobAbilityEnabled(false)
+            mob:setMobMod(xi.mobMod.NO_MOVE, 1)
+            mob:timer(3000, function(mobArg)
+                mobArg:entityAnimationPacket(xi.animationString.CAST_SUMMONER_STOP)
+                mobArg:setAutoAttackEnabled(true)
+                mobArg:setMobAbilityEnabled(true)
+                mobArg:setMobMod(xi.mobMod.NO_MOVE, 0)
+                mobArg:spawnPet()
+            end)
+        end
+    end
+
+    -- Bracelets activate after 80 seconds or 300 damage taken. They last for 30 seconds and give a 30% attack boost, +40 acc, and 33% haste.
+    local function activateBracelets(mob)
+        mob:setLocalVar('braceletActive', 1)
+        mob:setAnimationSub(2)
+        mob:addStatusEffect(xi.effect.ATTACK_BOOST, 300, 0, 30)
+        mob:addStatusEffect(xi.effect.HASTE, 3333, 0, 30) -- ~33% delay reduction
```

Comment:
When this says "delay reduction", do Aerns actually give you less TP on hitting you? If so, that's not haste but _true_ delay reduction

though it's likely to just be haste. Just making sure